### PR TITLE
PHP7 Update + option to deselect

### PR DESCRIPTION
**Pull request by animaux**

### CODE REVIEW

File: extension.meta.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+ <?xml version="1.0" encoding="UTF-8"?>
```
Comment (nitriques) [inline extension.meta.xml]: This might cause a problem... Can you remove the extra space please ?

File: extension.meta.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
 	</media>
 	<releases>
 		<release version="1.3.1" date="2018-03-21" min="2.3.x">
-		  - Add empty `<option>` to NOT select a file
+		  - Add empty `option` to NOT select a file
```
Comment (nitriques) [inline extension.meta.xml]: Since this is a new feature, it should be `1.4.0`

**Review by nitriques — CHANGES_REQUESTED** [review]
Thanks! Can you check the comments ?

**Comment by animaux** [comment]
Thanks for having a look! Not sure where that space came from … cleaned up code in a minute.

**Comment by nitriques** [comment]
np! Thanks !

**Pull request merged**